oak_launcher_utils/src/launcher/virtualized.rs

```diff
@@ -158,39 +152,51 @@ impl Instance {
 
         // Loading the application binary needs to happen before we start using microrpc over the
         // channel.
-        comms_host
+        host_socket
             .write(&(app_bytes.len() as u32).to_le_bytes())
             .expect("failed to send application binary length to enclave");
 
         // The kernel expects data to be transmitted in chunks of one page.
         let mut chunks = app_bytes.array_chunks::<PAGE_SIZE>();
         for chunk in chunks.by_ref() {
-            write_chunk(&mut comms_host, chunk)?;
+            Self::write_chunk(&mut host_socket, chunk)?;
         }
-        write_chunk(&mut comms_host, chunks.remainder())?;
+        Self::write_chunk(&mut host_socket, chunks.remainder())?;
 
         Ok(Self {
+            guest_console,
+            host_socket,
             instance,
-            console,
-            comms: comms_host,
         })
     }
+
+    /// Writes a chunk to a channel, and expects an acknowledgement containing the length of the
+    /// chunk.
+    fn write_chunk(channel: &mut dyn oak_channel::Channel, chunk: &[u8]) -> Result<()> {
```

Comment:
Same for this, it looks like it belongs to the top level scope as a stand-alone function.